interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -1,198 +1,263 @@
-      requestBody:
-        content:
-          multipart/form-data:
-            schema:
-              type: object
-              properties:
-                  files:
-                    type: array
-                    items:
-                      type: string
-                      format: binary
-        required: true
-      responses:
-        '200':
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/BackgroundTaskRunResponse'
-          description: >-
-            Returns task details of the task launched in the background to process KML files.
-        '400':
-          description: Failed to upload files.
+                  $ref: "#/components/schemas/TaskID"
+    /api/flight_records/json:
+        post:
+            description: "Upload flight record json files to be used to run tests."
+            requestBody:
+                content:
+                    multipart/form-data:
+                        schema:
+                            type: object
+                            properties:
+                                files:
+                                    type: array
+                                    items:
+                                        type: string
+                                        format: binary
+                required: true
+            responses:
+                "200":
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/FileUploadResponse"
+                    description: >-
+                        Returns Human readable message of uploaded file status.
+                "400":
+                    description: Failed to upload files.
+    /api/flight_records/kml:
+        post:
+            description: "Upload KML flight records that are processed to generate flight records json files."
+            requestBody:
+                content:
+                    multipart/form-data:
+                        schema:
+                            type: object
+                            properties:
+                                files:
+                                    type: array
+                                    items:
+                                        type: string
+                                        format: binary
+                required: true
+            responses:
+                "200":
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/BackgroundTaskRunResponse"
+                    description: >-
+                        Returns task details of the task launched in the background to process KML files.
+                "400":
+                    description: Failed to upload files.
+    /api/tests:
```

Comment:
We already have a resource collection named `test_runs` that we can POST to in order to create (trigger) a new test run to begin -- if we want to enumerate the resources in that collection, I would expect the endpoint to be `GET /api/test_runs`, which would return a list of the `test_run` resources.  The details for a particular resource would then be available at `GET /api/test_runs/{test_id}` (rather than `GET /api/tests/{test_id}`)